pkg/controller/controller.go

```diff
@@ -1199,3 +1205,664 @@ func (c *Controller) handleObject(obj interface{}) {
+			mysqldeployment, err = c.kubeclientset.AppsV1().Deployments(apimanager.Namespace).Create(mysql.MysqlDeployment(apimanager, "Pattern-2"))
+			if err != nil {
+				return err
+			}
+		}
+
+		// Get mysql service name using hardcoded value
+		mysqlservice, err := c.servicesLister.Services(apimanager.Namespace).Get(mysqlserviceName)
+
+		// If the resource doesn't exist, we'll create it
+		if errors.IsNotFound(err) {
+			mysqlservice, err = c.kubeclientset.CoreV1().Services(apimanager.Namespace).Create(mysql.MysqlService(apimanager))
+		} else {
+			fmt.Println("Mysql Service is already available. [Service name] ,", mysqlservice)
+		}
+
+		for mysqldeployment.Status.ReadyReplicas == 0 {
+			time.Sleep(5 * time.Second)
+			mysqldeployment, err = c.deploymentsLister.Deployments(apimanager.Namespace).Get(mysqldeploymentName)
+		}
+	}
+
+	// Get analytics dashboard deployment name using hardcoded value
+	dashdeployment, err := c.deploymentsLister.Deployments(apimanager.Namespace).Get(dashboardDeploymentName)
+	if enableAnalytics {
+		// If the resource doesn't exist, we'll create it
+		if errors.IsNotFound(err) {
+			y := pattern2.AssignApimAnalyticsDashboardConfigMapValues(apimanager, configmap, dashnum)
+
+			dashdeployment, err = c.kubeclientset.AppsV1().Deployments(apimanager.Namespace).Create(pattern2.DashboardDeployment(apimanager, y, dashnum))
+			if err != nil {
+				return err
+			}
+		}
+
+		// Get analytics dashboard service name using hardcoded value
+		dashservice, err := c.servicesLister.Services(apimanager.Namespace).Get(dashboardServiceName)
+		// If the resource doesn't exist, we'll create it
+		if errors.IsNotFound(err) {
+			dashservice, err = c.kubeclientset.CoreV1().Services(apimanager.Namespace).Create(pattern2.DashboardService(apimanager))
+		} else {
+			fmt.Println("Dash Service is already available. [Service name] ,", dashservice)
+		}
+
+		// Get ingress name using hardcoded value
+		dashIngress, err := c.ingressLister.Ingresses(apimanager.Namespace).Get(dashIngressName)
+		// If resource doesn't exist, we'll create it
+		if errors.IsNotFound(err) {
+			dashIngress, err = c.kubeclientset.ExtensionsV1beta1().Ingresses(apimanager.Namespace).Create(pattern2.DashboardIngress(apimanager))
+		} else {
+			fmt.Println("Dash Ingress is already available. [Ingress name] ,", dashIngress)
+		}
+
+	}
+
+	// Get analytics worker deployment name using hardcoded value
+
+	// workerdeployment, err := c.deploymentsLister.Deployments(apimanager.Namespace).Get(workerDeploymentName)
+	workerdeployment, err := c.statefulSetsLister.StatefulSets(apimanager.Namespace).Get(workerDeploymentName)
+	if enableAnalytics {
+		// If the resource doesn't exist, we'll create it
+		if errors.IsNotFound(err) {
+			y := pattern2.AssignApimAnalyticsWorkerConfigMapValues(apimanager, configmap, worknum)
+
+			// workerdeployment, err = c.kubeclientset.AppsV1().Deployments(apimanager.Namespace).Create(pattern1.WorkerDeployment(apimanager, y, worknum))
```

Comment:
remove the comment

Reply:
addressed via https://github.com/wso2/k8s-wso2am-operator/pull/89/commits/788038d4b86839a0ede2e666405f0c4f501c0dbf